### PR TITLE
 - minor improvements improving the Importer robustness

### PR DESCRIPTION
**Pull request by afabiani**
Related to JIRA GEOS-7426

### CODE REVIEW

File: src/extension/importer/core/src/main/java/org/geoserver/importer/FeatureDataConverter.java [hunk 1]
```diff
@@ -187,6 +187,15 @@ String attName(String name) {
         }
     };
 
+    private static boolean containsAttribute(SimpleFeature ft, String attName) {
```
Comment (aaime) [inline src/extension/importer/core/src/main/java/org/geoserver/importer/FeatureDataConverter.java]: Why static?

File: src/extension/importer/core/src/main/java/org/geoserver/importer/FeatureDataConverter.java [hunk 2]
```diff
@@ -187,6 +187,15 @@ String attName(String name) {
...
+        for (AttributeDescriptor att : ft.getType().getAttributeDescriptors()) {
```
Comment (aaime) [inline src/extension/importer/core/src/main/java/org/geoserver/importer/FeatureDataConverter.java]: Err... if I understand what this does, wouldn't it be the same to check if obj is null or not?

**Comment by aaime** [comment]
Copyright headers have not been updated, see https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md#commit-guidelines

**Comment by afabiani** [comment]
Fixed

Best Regards,
Alessio Fabiani.

==
GeoServer Professional Services from the experts!
Visit http://goo.gl/it488V for more information.
==

Ing. Alessio Fabiani
@alfa7691
Founder/Technical Lead

GeoSolutions S.A.S.
Via di Montramito 3/A
55054  Massarosa (LU)
Italy
phone: +39 0584 962313
fax:     +39 0584 1660272
mob:   +39 333 8128928

http://www.geo-solutions.it
http://twitter.com/geosolutions_it

-------------------------------------------------------

*AVVERTENZE AI SENSI DEL D.Lgs. 196/2003*

Le informazioni contenute in questo messaggio di posta elettronica e/o
nel/i file/s allegato/i sono da considerarsi strettamente riservate. Il
loro utilizzo è consentito esclusivamente al destinatario del messaggio,
per le finalità indicate nel messaggio stesso. Qualora riceviate questo
messaggio senza esserne il destinatario, Vi preghiamo cortesemente di
darcene notizia via e-mail e di procedere alla distruzione del messaggio
stesso, cancellandolo dal Vostro sistema. Conservare il messaggio stesso,
divulgarlo anche in parte, distribuirlo ad altri soggetti, copiarlo, od
utilizzarlo per finalità diverse, costituisce comportamento contrario ai
principi dettati dal D.Lgs. 196/2003.



The information in this message and/or attachments, is intended solely for
the attention and use of the named addressee(s) and may be confidential or
proprietary in nature or covered by the provisions of privacy act
(Legislative Decree June, 30 2003, no.196 - Italy's New Data Protection
Code).Any use not in accord with its purpose, any disclosure, reproduction,
copying, distribution, or either dissemination, either whole or partial, is
strictly forbidden except previous formal approval of the named
addressee(s). If you are not the intended recipient, please contact
immediately the sender by telephone, fax or e-mail and delete the
information in this message that has been received in error. The sender
does not give any warranty or accept liability as the content, accuracy or
completeness of sent messages and accepts no responsibility  for changes
made after they were sent or for other risks which arise as a result of
e-mail transmission, viruses, etc.

---------------------------------------------------------------------

**Comment by aaime** [comment]
Manually merged and backported to 2.8.x

**Pull request closed without merging**